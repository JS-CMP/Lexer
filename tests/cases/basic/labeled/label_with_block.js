for (;;) {
    console.log("loop");
    myLabel: {
        console.log("start");
        break myLabel;
        console.log("end");
    }
}

