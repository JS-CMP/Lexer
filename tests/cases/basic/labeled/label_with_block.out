#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {
    for (; ; ) 
    {
        console[u"log"](JS::Any(u"loop"));
myLabel:
        
        {
            console[u"log"](JS::Any(u"start"));
            break /* label: ; JS_CMP does not support labeled break for now */;
            console[u"log"](JS::Any(u"end"));
        }
    }
    return 0;
}
